built-in-policies/policySetDefinitions/Kubernetes/Kubernetes_PSPRestrictedStandard.json

```diff
@@ -137,6 +137,7 @@
           },
           "allowedVolumeTypes": {
             "value": [
+              "csi",
```

Comment:
@arturs-savickis The built-in policy definitions hardcodes the volume types as per the pod security standards specified here https://v1-18.docs.kubernetes.io/docs/concepts/security/pod-security-standards/#policy-instantiation. 
To add new values, please create a custom initiative.